products/ocp4/profiles/cis.profile

```diff
@@ -30,6 +30,7 @@ selections:
     - cis_ocp_1_4_0:all
     ### Variables
     - var_openshift_audit_profile=WriteRequestBodies
+    - var_event_record_qps=50
```

Comment:
This is a bit superfluous, since the default value is being changed to `50`.
But this is more explicit though.